### DISCUSSION TITLE
GUI CHIA

### DISCUSSION THREAD

**Discussion by TomaszWM** [Chia-Network/chia-blockchain#8305] [Support]
On gui chia how I  can delete the unused pool contracts ? For example  Red Reptile on  the picture belong.
![SoacePool Change](https://user-images.githubusercontent.com/84909392/131833628-2ca239ea-e3ff-45f8-a35a-7bdfad59507e.JPG)

**Comment by Jacek-ghub** [comment]
I don't think that you can do that using chia UI, as it is too primitive.  Although, you can remove it from the config file.  Take a look at that video:

https://youtu.be/F0VHlvvf8fE?t=102

**Reply by loppefaaret** [reply]
in a future update, we have planned for users to be able to hide unwanted plotNFT's, but it is not possible currently, from an official standpoint

**Comment by Jacek-ghub** [comment]
@loppefaaret 
In such a case, if we remove pool info from the config.yaml, isn't it the same as removing a pool or hiding it?  Or are we causing a potential instability in the setup by doing that?

**Reply by loppefaaret** [reply]
if there are no plots tied to it, i _personally_ can't see any problems with it - if it _does_ have plots tied to it, i'd imagine there could very well be problems correctly farming those plots

**Comment by Jacek-ghub** [comment]
As plots are bound to the (local, or rather UI) pool, I would also imagine that removing such pool from config.yaml will render those plots orphaned.  Or rather those pools belong to a wallet, and info in config.yaml is just a cached info?

Although, if config.yaml is the only place that keeps a given pool info, that may also imply that if someone restores that pool info in config, those plots would get a second life.  Still, removing a pool with plots is not really a good idea what potentially implies that one should keep a backup of config.yaml just for that reason.

**Reply by loppefaaret** [reply]
there is some new stuff in %appdata% atleast on windows, im not entirely proficient in that area yet to speak of it

**Reply by Jacek-ghub** [reply]
I guess, my point was that if pool info is only stored locally, in case of a full-node crash (HD failure), the farm pool data is lost, as such all plots are rendered no good anymore, unless one have a backup of config.yaml.  If part of that info is stored in other local files, it would be nice to get a guidance on what needs to be fully backed up (I know - everything, but ...).

**Reply by loppefaaret** [reply]
my best suggestion, is to let the wallet go through the sync, and generate the entries again. that would be the official way of doing it.
that avoids any potential misaligned data in the entry at any point

**Reply by Jacek-ghub** [reply]
My understanding was that when you generate a pool, you always get a new contract (i.e., the pool is unique).  As such, you cannot re-generate those pools you lost due to a HD crash.  They need to be either restored by the wallet/blockchain (if stored there) or brought back from backups (if only stored locally).

**Reply by loppefaaret** [reply]
apart from the plots themselfs, everything else can be restored from the chain and with your 24 words - what i mean is that you might have a backup from 14 days ago, but did something last week - and then if you choose to manually restore the pol list in config.yaml - you might have incorrect data in the entry, and i cant say how the client reacts if it meets unexpected values in there...  hence why i suggest sticking to restoring the wallet from before the plotNFT creation - personally, i got a wallet db backup the day before i created my first plotNFT - so i can skip 500k blocks of wallet syncing, should i ever need to

**Reply by Jacek-ghub** [reply]
Thank you!

**Reply by Jacek-ghub** [reply]
I also used before backed up wallet db to speed up syncing, although just for a new wallet creation testing - worked great.

With that said, would it be possible to do a live backups of walled db as a client enhancement?  For instance, specify duration/location/retention count for those backups in config.yaml.  This way, it would not be a manual task that requires stopping the client, plus your backup would always be close to the latest.

**Reply by loppefaaret** [reply]
currently, both the node and wallet DB needs to be backed up while being offlined - wether there is any plans for it, not as far as i know, and i do not know about any ongoing work - there is some wallet things planned - but when is up in the air
as always, we do have a public available trello board with lots of plans pinned on it
https://trello.com/b/ZuNx7sET/engineering-core
though we at times do could be better at keeping it up to date - just that the devs are just kinda deepdiving straight into the next issue constantly

**Reply by Jacek-ghub** [reply]
Maybe devs can look into master/slave libs (e.g., LiteReplica, or sqlite2-sync)?  Looks like not that much work to enable the client to be the master, and gives an option to implement (if desired) a slave on another box.  With such design, you can always pause the slave to get up-to-date backup (manually or through a script), whether for testing, doing backups, or to speed up restoring the client.  

I am not a db expert, but implemented it some time ago for MySQL, and it worked great.  Not really that much work needed, but huge benefit of having hot backups.

**Comment by BrandtH22** [comment]
@TomaszWM , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
